src/engraving/rendering/dev/beamlayout.cpp

```diff
@@ -1011,6 +1012,7 @@ void BeamLayout::checkCrossPosAndStemConsistency(Beam* beam, LayoutContext& ctx)
     if (inconsistencyFound) {
         BeamLayout::layout(beam, ctx);
     }
+    SystemLayout::layoutSystemElements(beam->system(), ctx);
```

Comment:
Urgh, we must have fixed this same bug at least 20 times before, always because of ledger lines and the fact that they are deleted at every layout. We need to change how ledger lines are created, at some point, we've known this for a while. And this may be the point that we can't postpone that job anymore. Cause we really can't solve it like this unfortunately, layoutSystemElements is _extremely_ expensive. We need to find a different way